src/platform/code-editor/code-editor.component.ts

```diff
@@ -591,6 +591,58 @@ export class TdCodeEditorComponent implements OnInit, AfterViewInit, ControlValu
     return loadPromise;
   }
 
+  showFullScreenEditor(): void {
```

Comment:
It would be better to be more explicit with this method so can see more clearly what is going on, and use comments as well.  Something like this:

```
showFullScreenEditor(): void {
    const codeEditorElement: HTMLDivElement = this._editorContainer.nativeElement as HTMLDivElement;

    if (codeEditorElement.requestFullscreen) {
      codeEditorElement.requestFullscreen();
    // Chrome / Edge / Safari
    } else if (codeEditorElement.webkitRequestFullscreen) {
      codeEditorElement.webkitRequestFullscreen();
    // IE
    } else if ((<any>codeEditorElement).msRequestFullscreen) {
      (<any>codeEditorElement).msRequestFullscreen();
    // Firefox
    } else if ((<any>codeEditorElement).mozRequestFullScreen) {
      (<any>codeEditorElement).mozRequestFullScreen();
    }
  }
```

Reply:
I wanted to avoid multiple if-else conditions so i added map here.
Will add proper comments.